.eslintcache

```diff
@@ -1 +1 @@
d=\"insta\">\n              <h2>Check out</h2>\n              <h2 id=\"insta-link\">\n                Trigo on{\" \"}\n                <a\n                  href=\"https://www.instagram.com/trigobakeryinc/\"\n                  target=\"_blank\"\n                  rel=\"noreferrer\"\n                >\n                  instagram...\n                </a>\n              </h2>\n            </div>\n          </div>\n        </div>\n        <div className=\"section white\">\n          <div className=\"row\">\n            <div id=\"carousel\">\n              <img src={carousel1} alt=\"pastries\" />\n              <img src={carousel2} alt=\"pastries\" />\n              <img src={carousel3} alt=\"pastries\" />\n              <img src={carousel4} alt=\"pastries\" />\n            </div>\n          </div>\n        </div>\n      </div>\n\n      {/* <Header />\n      <HeaderImage />\n      <div className=\"landing-section color\">\n        <div className=\"content\" id=\"bread-row\">\n          <div className=\"column\" id=\"bread-description\">\n            <img src={grains} alt=\"grains\" />\n            <h1>Bread</h1>\n            <p>\n              Bread may be leavened by naturally occurring microbes, chemicals,\n              industrially produced yeast, or high-pressure aeration. In many\n              countries, commercial bread often contains additives to improve\n              flavor, texture, color, shelf life, nutrition, and ease of\n              production.\n            </p>\n          </div>\n          <div className=\"column right\" id=\"ingredients\">\n            \n          </div>\n        </div>\n      </div>\n\n      <div className=\"landing-section white\">\n        <Footer landing={true} />\n      </div> */}\n    </div>\n  );\n};\n\nexport default LandingPage;\n","/Users/jaeyoung/personal/TrigoBakery/src/components/UI/NavBar/NavBar.js",[],"/Users/jaeyoung/personal/TrigoBakery/src/components/Footer/Footer.js",[],"/Users/jaeyoung/personal/TrigoBakery/src/components/Header/Header.js",[],["65","66"],"/Users/jaeyoung/personal/TrigoBakery/src/components/UI/HeaderButton/HeaderButton.js",[],"/Users/jaeyoung/personal/TrigoBakery/src/components/Header/HeaderImage/HeaderImage.js",[],"/Users/jaeyoung/personal/TrigoBakery/src/components/ProductCard/ProductCard.js",[],"/Users/jaeyoung/personal/TrigoBakery/src/components/MoreCard/MoreCard.js",[],{"ruleId":"67","replacedBy":"68"},{"ruleId":"69","replacedBy":"70"},{"ruleId":"71","severity":1,"message":"72","line":1,"column":17,"nodeType":"73","messageId":"74","endLine":1,"endColumn":26},{"ruleId":"71","severity":1,"message":"75","line":17,"column":8,"nodeType":"73","messageId":"74","endLine":17,"endColumn":14},{"ruleId":"67","replacedBy":"76"},{"ruleId":"69","replacedBy":"77"},"no-native-reassign",["78"],"no-negated-in-lhs",["79"],"no-unused-vars","'useEffect' is defined but never used.","Identifier","unusedVar","'Footer' is defined but never used.",["78"],["79"],"no-global-assign","no-unsafe-negation"]
```

Comment:
don't commit cache files

Reply:
.eslintcache was placed in the .gitignore file, should i delete .eslintcache instead?

Reply:
you can delete it, or not. Just when you add your files, be sure not to include it in any commits